testsuite/tests/lib-reachable/testopt.ml

```diff
@@ -0,0 +1,82 @@
+(* TEST
+   * native
+*)
+
+(* todo: TEXSTXX
+
+include dynlink
+libraries = ""
+files = "plugin.ml testopt.ml"
+
+* native
+** shared-libraries
+*** setup-ocamlopt.opt-build-env
+**** ocamlc.opt
+program = "${test_build_directory}/testopt.opt"
+flags = "-linkall"
+include ocamlcommon
+libraries += "dynlink"
+all_modules = "testopt.ml"
+***** run
+******check-program-output
+*)
+let _ =
+  let test_src_dir = Sys.getenv("test_source_directory") in
+  let flambda = try Sys.getenv("CONFIG_ARG") = "--enable-flambda" with Not_found -> false in
```

Comment:
This is a Travis-specific variable, so it can't go in a test. You already include ocamlcommon above, so you have access to `Config.flambda`